src/content/peerconnection/trickle-ice/js/main.js

```diff
@@ -128,29 +140,26 @@ function noDescription(error) {
 function parseCandidate(text) {
   var candidateStr = 'candidate:';
   var pos = text.indexOf(candidateStr) + candidateStr.length;
-  var fields = text.substr(pos).split(' ');
+  var [foundation, component, protocol, priority, address, port, , type] = text.substr(pos).split(' ');
```

Comment:
maybe we should wait until we have https://github.com/webrtc/adapter/pull/527 ;-)